src/network/tarcap/packets_handlers/common/packet_handler.cpp

```diff
@@ -64,7 +64,7 @@ bool PacketHandler::sealAndSend(const dev::p2p::NodeID& nodeID, SubprotocolPacke
     peer = peers_state_->getPendingPeer(nodeID);
 
     if (!peer) {
-      LOG(log_wr_) << "sealAndSend failed to find peer";
+      LOG(log_er_) << "sealAndSend failed to find peer";
```

Comment:
why error? peer could disconnect meanwhile, so it is not an error for this node but for his more likely

Reply:
I will approve the PR but besides here I also see a couple of other places where errors are logged for a case when peer might have just disconnected

Reply:
I want to see if it happens a lot, if it is warning we will never know as we wont have enabled channels for every packet handler

Reply:
@mfrankovi If those errors should not be errors then do not approve such PR and write comments. If you both think this should be warnng I will change it

Reply:
changed to wr